src/components/IconCustomizer.svelte

```
@@ -1,63 +1,46 @@
-			})();
+	$: if ($customizedIcon?.userIconId && $customizedIcon.userIconId !== UUID.empty) {
+		// Has userIconId and isn't empty uuid?
+		if (isContentTypeSvg($customizedIcon.contentType) && $customizedIcon.svgContent) {
+			customizedIcon.selectSvgIcon($customizedIcon.svgContent);
+		} else {
+			// _.flow(IconService.mkIconUrl, customizedIcon.selectImageIcon)($customizedIcon.originalIconId);
+			throw new Error('User icon is not an SVG. Only SVGs are supported for now.');
 		}
+	}
+
+	$: if ($customizedIcon?.iconId && (!$customizedIcon.userIconId || $customizedIcon.userIconId === UUID.empty)) {
```

ESLint

[error] Properties of objects and arrays are not reactive unless in runes mode. Changes to this property will not cause the reactive statement to update(reactive_declaration_non_reactive_property)

(svelte/valid-compile)